src/styles/colors.js

```diff
@@ -65,11 +67,53 @@ const gris80 = '#858585';
 const gris90 = '#5E5E5E';
 const gris100 = '#383838';
 
+// #7976B5
+const wildBlueHsl = {
+  hue: 243,
+  saturation: 30,
+  lightness: 59,
+};
+// #FFB5AF
+const salmonHsl = {
+  hue: 5,
+  saturation: 100,
+  lightness: 84,
+};
+// #D5BAF8
+const mauve30Hsl = {
+  hue: 266,
+  saturation: 82,
+  lightness: 85,
+};
+// #D4FCF5
+const turquoise20hsl = {
+  hue: 170,
+  saturation: 87,
+  lightness: 91,
+};
+// #FDF9C4
+const jaune30hsl = {
+  hue: 56,
+  saturation: 93,
+  lightness: 88,
+};
+// #CCEAFF
+const ciel20hsl = {
+  hue: 205,
+  saturation: 100,
+  lightness: 90,
+};
+// #FFD6F4
+const rose20hsl = {
+  hue: 316,
+  saturation: 100,
+  lightness: 92,
+};
 // #000CA0
 const bleu80hsl = {
-  hue: 236,
-  saturation: 100,
-  lightness: 31,
+  hue: 243,
```

Comment:
À moins d'un GO officiel, je ne modifierais pas maintenant complètement le bleu80 pour la raison que j'évoquais dans mon commentaire sur [Slack](https://waq2015.slack.com/archives/G01FUNCKZRN/p1678799128165869?thread_ts=1678584948.046699&cid=G01FUNCKZRN) soit de garder une certaine homogénéité avec le reste des visuels du WAQ et d'aller au rythme des changements apportés par le comité Design. J'équilibrerais plutôt avec le bleu90 afin de l'estomper à travers le site.